src/common/metric.tsx

```diff
@@ -1,3 +1,4 @@
+import { Metric } from 'common/metricEnum';
```

Comment:
We could re-export `Metric` from this file, so we don't have to change the imports in all the other files changed in this branch